File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {

       int temp=x;
       String s="";
       s=s+x;
       char ch[]=s.toCharArray();
       int f=0;
       int l=ch.length-1;
       while(f<l)
       {
           if(ch[f]!=ch[l])
           {
               return false;
           }
           f++;
           l--;
       }
       return true;
    }      
    
}

File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        
        int rev=0,temp=x;
        while(x>0)
        {
            int d=x%10;
            rev=rev*10+d;
            x=x/10;
        }
        return(rev==temp);
    }
}